### PR TITLE
Haley Holmes - Image Analysis Pipeline Edit

### PR DESCRIPTION
**Pull request by haley-holmes**
Edits made on 01/04/21 by Haley Holmes. Some paragraph reformatting, small grammar and spelling fixes. finFindR needs a subsection and navigational sidebar link as it is the only program excluded on this page that appears in the ID "ecosystem" diagram (the one with zebras and whales). Unfortunately I have no knowledge on this program and therefore cannot aid in this matter.

### CODE REVIEW

File: docs/researchers/ia_pipeline.md [hunk 1]
```diff
@@ -48,82 +52,82 @@ Even machine learning makes mistakes. If detection doesn't find an animal, [Wild
...
 
 ![IA_pipeline](../../static/img/IA_pipeline.png)
 
-*Wildbook has a plug-and-play, configurable computer vision pipeline that allows Annotations for different species and of body parts to be  simultaneously routed to one or more ID algorithms.*
+*Wildbook has a plug-and-play, configurable computer vision pipeline that allows annotations for different species and of body parts to be simultaneously routed to one or more ID algorithms.*
 
-After detection, one or more ID algorithms are run, looking for similar features between a new Annotation and a list of "match against" Annotations already in the database. Scores from the query that match the same individual are accumulated to produce a single potential score for each animal. The animals in the database are then ranked by their accumulated scores.
+After detection, one or more ID algorithms are run, looking for similar features between a new anotation and a list of "match against" anotations already in the database. Scores from the query that match the same individual are accumulated to produce a single potential score for each animal. The animals in the database are then ranked by their accumulated scores.
```
Comment (TanyaStere42) [inline docs/researchers/ia_pipeline.md]: typo: annotation, not anotation

File: docs/researchers/ia_pipeline.md [hunk 2]
```diff
@@ -48,82 +52,82 @@ Even machine learning makes mistakes. If detection doesn't find an animal, [Wild
...
 
-Example Wildbook species: Giant manta rays, humpback whale flukes, orcas, right whales.
+Example Wildbook Species: Giant manta rays, humpback whale flukes, orcas, right whales
```
Comment (TanyaStere42) [inline docs/researchers/ia_pipeline.md:73]: why is "Species" capitalized?

File: docs/researchers/ia_pipeline.md [hunk 3]
```diff
@@ -48,82 +52,82 @@ Even machine learning makes mistakes. If detection doesn't find an animal, [Wild
...
 
-PIE paper link:https://arxiv.org/pdf/1902.10847.pdf
+PIE Paper Link:https://arxiv.org/pdf/1902.10847.pdf
```
Comment (TanyaStere42) [inline docs/researchers/ia_pipeline.md]: Why is "Paper Link" capitalized?

File: docs/researchers/ia_pipeline.md [hunk 4]
```diff
@@ -48,82 +52,82 @@ Even machine learning makes mistakes. If detection doesn't find an animal, [Wild
...
+Kaggle7 is a deep learning approach to identifying individual humpback whale flukes based on one of the winning entries from a Kaggle competition. Kaggle7 is highly effective at identifying individual humpback whales by their natural fluke patterning, however it must be re-trained when new whale individuals are added to a catalog. Like many deep learning competition winners, this algorithm is highly optimized for its task and cannot be cross-applied to other wildlife challenges. It is also highly subjective to the quality of the data in a humpback catalog, with performance degrading significantly (due to machine-learned errors) if the catalog is not already very well curated.
```
Comment (TanyaStere42) [inline docs/researchers/ia_pipeline.md]: adjust retraining language to match Deepsense language

**Review by TanyaStere42 — CHANGES_REQUESTED** [review]
Looks great overall, couple of questions where I just want to confirm it was a deliberate change with an understanding of the language, couple of requests for changes. LOVE the additions to the deepsense description. You added very little, but made it very clear what is going on in terms of retraining, great job!

File: docs/researchers/ia_pipeline.md (hunk 2 above)
Comment (haley-holmes) [reply docs/researchers/ia_pipeline.md:73]: I capitalized Species for the aesthetic value for the website, if you would like it changed back that's no problem! :)

File: docs/researchers/ia_pipeline.md (hunk 3 above)
Comment (haley-holmes) [reply docs/researchers/ia_pipeline.md]: I also capitalized Paper Link for aesthetic value and can easily be reverted.

File: docs/researchers/ia_pipeline.md [hunk 5]
```diff
@@ -58,7 +58,7 @@ The second major computer vision step is identification, which assigns a name la
 
 *Wildbook has a plug-and-play, configurable computer vision pipeline that allows annotations for different species and of body parts to be simultaneously routed to one or more ID algorithms.*
 
-After detection, one or more ID algorithms are run, looking for similar features between a new anotation and a list of "match against" anotations already in the database. Scores from the query that match the same individual are accumulated to produce a single potential score for each animal. The animals in the database are then ranked by their accumulated scores.
+After detection, one or more ID algorithms are run, looking for similar features between a new annotation and a list of "match against" anotations already in the database. Scores from the query that match the same individual are accumulated to produce a single potential score for each animal. The animals in the database are then ranked by their accumulated scores.
```
Comment (TanyaStere42) [inline docs/researchers/ia_pipeline.md]: there's another case of "anotation" instead of "annotation" here

**Review by TanyaStere42 — CHANGES_REQUESTED** [review]
just a typo got missed, the rest looks great

**Review by TanyaStere42 — APPROVED** [review]

**Pull request merged**